src/components/FilterTags/FilterTags.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { useRef, useEffect, useLayoutEffect, useState, Children } from 'react';
+import PropTypes from 'prop-types';
+import { OverflowMenuItem, OverflowMenu } from 'carbon-components-react';
+import { Close16 } from '@carbon/icons-react';
+import classnames from 'classnames';
+
+import { settings } from '../../constants/Settings';
+import { useResize } from '../../internal/UseResizeObserver';
+
+const { prefix, iotPrefix } = settings;
+/* eslint-disable-next-line react/prop-types */
+const DefaultWrapper = React.forwardRef(({ children, ...props }, ref) => {
+  return (
+    <div {...props} ref={ref}>
+      {children}
+    </div>
+  );
+});
+/* eslint-disable-next-line react/prop-types */
+const OverflowTag = ({ children }) => (
+  <span>
+    {children} <Close16 />
+  </span>
+);
+
+const FilterTags = ({ children, hasOverflow, id, tagContainer }) => {
+  const TagContainer = tagContainer || DefaultWrapper;
+  const overFlowContainerRef = useRef(null);
+  useResize(overFlowContainerRef);
+  const childrenItems = Children.map(children, child => child);
+  const breakingWidth = useRef([]);
+  const [overflowItems, setOverflowItems] = useState([]);
+  const [visibleItems, setVisibleItems] = useState(childrenItems);
+
+  useEffect(
+    () => {
+      /* istanbul ignore else */
```

Comment:
Is there a reason to not get coverage here? Its a viable use-case that Overflow is not set

Reply:
Just realized you replied to similar comments from @tay1orjones . The `else` branch is here that this useEffect doesn't do anything so I think it should be tested for (which it looks it will be covered by your tests)

Reply:
There is no else logic which is what the coverage is complaining about. So it falsely reports a lack of coverage..